src/domain/individualCustomer/IndividualCustomerPageContainer.tsx

```diff
@@ -1,23 +1,20 @@
 import React from 'react';
 import { useParams } from 'react-router-dom';
 import { useQuery } from '@apollo/react-hooks';
-import { useTranslation } from 'react-i18next';
 
 import IndividualCustomerPage from './individualCustomerPage/IndividualCustomerPage';
 import { INDIVIDUAL_CUSTOMER_QUERY } from './queries';
 import { INDIVIDUAL_CUSTOMER } from './__generated__/INDIVIDUAL_CUSTOMER';
 import Card from '../../common/card/Card';
-import Paragraph from '../../common/paragraph/Paragraph';
-import LabelValuePair from '../../common/labelValuePair/LabelValuePair';
-import Text from '../../common/text/Text';
+import BillsCard from './billsCard/BillsCard';
```

Comment:
CustomerInfoCard and BillsCard look great and IndividualCustomerPageContainer.tsx is improved a lot.